### PR TITLE
Allow the Reply-to header of e-mail contain more than one e-mail address

### PR DESCRIPTION
**Pull request by rturella3**
The [RFC 2822](https://www.rfc-editor.org/rfc/rfc2822.txt) says:

> an optional reply-to field MAY also be
> included, which contains the field name "Reply-To" and a
> comma-separated list of one or more addresses.
> ...
> reply-to        =       "Reply-To:" address-list CRLF

In CakePHP, if you try to set more than one reply-to, an InvalidArgumentException is thrown.

This pull request is to allow more than 1 e-mail address as reply-to.

### CODE REVIEW

File: src/Mailer/Message.php [hunk 1]
```diff
@@ -394,6 +394,19 @@ public function getReplyTo(): array
...
 
+    /**
+     * Add "Reply-To" address.
+     *
+     * @param string|array $email Null to get, String with email,
+     *   Array with email as key, name as value or email as value (without name)
+     * @param string|null $name Name
+     * @return $this
+     */
+    public function addReplyTo($email, ?string $name = null)
```
Comment (ADmad) [inline src/Mailer/Message.php:405]: We don't add new methods in patch releases. Other changes seem fine for `master`. So you should make a separate PR targeting `4.next` to add this method after the other changes are merged.

**Comment by rturella3** [comment]
@ADmad, thanks for review.

I'm new to git, can you help me in how do I proceed? Do I remove the function (and the tests that use it) and make a new commit in my fork? If I do this, will the pull request "self-update"?

**Comment by ADmad** [comment]
@rturella3 Yes you can remove the function and it's test and make a new commit. Then just push the branch and the pull request will auto update.

**Review by ADmad — APPROVED** [review]
